assets/queries/terraform/azure/php_version_not_latest_when_running_web_app/query.rego

```diff
@@ -0,0 +1,51 @@
+package Cx
+
+import data.generic.common as common_lib
+
+# for deprecated version (before AzureRM 3.0)
+CxPolicy[result] {
+	resource := input.document[i].resource.azurerm_app_service[name]
+	php_version := resource.site_config.php_version
+    to_number(php_version) != 7.4
+    
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("azurerm_app_service[%s].site_config.php_version", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "for the attribute 'php_version' to be the lastest version (7.4)",
+		"keyActualValue": "'php_version' is not the lastest version (7.4)",
+		"searchLine": common_lib.build_search_line(["resource", "azurerm_app_service", name, "site_config", "php_version"], []),
+	}
+}
+
+# After 3.0, for windows
+CxPolicy[result] {
+	resource := input.document[i].resource.azurerm_windows_web_app[name]
+    php_version := resource.site_config.application_stack.php_version
+	php_version != "v7.4"
```

Comment:
As mentioned in the CIS Microsoft Azure Foundations Benchmark v1.4.0 (item 9.6), the PHP version should be the latest available stable release that is PHP 8.0

```suggestion
    php_version != "v8.0"
```

Reply:
Actually that 8.1 (:
https://www.php.net/downloads

Reply:
Thank you, @kaplanlior! 🚀